### PR TITLE
Double-side loaded transmission line model and multi-node subsystems

### PR DESCRIPTION
**Pull request by quantum-quant**
### What are the issues this pull addresses (issue numbers / links)?
* Implemented double-side loaded TL resonator model
* New TL model nicely resolves to single-side loaded and/or single-side shorted models under approriate boundary conditions
* Generalize multi-node subsystems which were necessary for the double-sided TL
model

### Did you add tests to cover your changes (yes/no)?
* Checked and verified that the double-side loaded TL model can recover single-side loaded results under approriate bouonary conditions
* Check that new changes do not break existing tutorials/applications

### Did you update the documentation accordingly (yes/no)?
yes

### Did you read the CONTRIBUTING document (yes/no)?
yes

### CODE REVIEW

**Comment by ThomasGM4** [comment]
I presume want to merge in #790 first?

File: qiskit_metal/analyses/quantization/lom_core_analysis.py [hunk 1]
```diff
@@ -63,52 +63,97 @@ class CouplingType:
...
         vp (float): phase velocity in m/s
         Z0 (float): characteristic impedance of the TL in ohm
+        cap_loading (dict): a dictionary of the loading capacitors; the keys
+            are the names of the nodes; the values the capacitances in fF
         shorted (boolean): default false; true if the other end of the TL is shorted false otherwise
 
     Returns:
         [type]: [description]
     """
+    _POS_INFTY = 1e30
```
Comment (zlatko-minev) [inline qiskit_metal/analyses/quantization/lom_core_analysis.py:83]: What about different systems?

**Review by zlatko-minev — CHANGES_REQUESTED** [review]
Add units

File: qiskit_metal/analyses/quantization/lom_core_analysis.py [hunk 2]
```diff
@@ -1235,25 +1322,33 @@ def compute_gs(self, coupling_type: str = CouplingType.CAPACITIVE):
...
+                            if node1 == node2:
+                                continue
+                            idx1 = self.node_index(node1)
+                            idx2 = self.node_index(node2)
+                            c_i = c_inv_k[idx1, idx2]
+                            if c_i == 0:
+                                continue
+                            Q_zpf_1 = sub1.h_params[node1]['Q_zpf']
+                            Q_zpf_2 = sub2.h_params[node2]['Q_zpf']
+                            g[idx1, idx2] = c_i * Q_zpf_1 * Q_zpf_2 * \
```
[the remark this answers is not in the document]
Comment (quantum-quant) [reply qiskit_metal/analyses/quantization/lom_core_analysis.py:1347]: added unit comment in docstring

File: qiskit_metal/analyses/quantization/lom_core_analysis.py [hunk 3]
```diff
@@ -1321,6 +1333,8 @@ def compute_gs(self, coupling_type: str = CouplingType.CAPACITIVE):
         """compute g matrices from reduced C inverse matrix C^{-1}_{k} and reduced L inverse matrix L^{-1}_{k}
         and zero point flucuations (Q_zpf, Phi_zpf)
 
+        Note that the computed g's are in MHz
```
Comment (zlatko-minev) [inline qiskit_metal/analyses/quantization/lom_core_analysis.py:1336]: Nice

**Review by zlatko-minev — APPROVED** [review]
